Client/Extensions/String+Extension.swift

```diff
@@ -18,6 +18,19 @@ extension String {
         return self.attributedText(boldIn: range, font: font)
     }
 
+    /// Returns an attributed string in which the first occurrence of the given
+    /// substrings is bold.
+    /// - Parameters:
+    ///     - boldStrings: the substrings that should be bold
+    ///     - font: font for entire string, part of string will be converted to bold version of this font
+    func attributedText(boldStrings: [String], font: UIFont) -> NSAttributedString {
```

Comment:
So there are some methods added in [this PR](https://github.com/mozilla-mobile/firefox-ios/pull/16339/files#diff-5f9f053e04df4b650364f637a75f7819a17fa55a94f846f40827e0d0b194daf3), can we have some common ground there? Would be ideal to have a discussion there so we don't have three new bold string methods for attributed text that does similar things (on top of one that already existed), but just different enough that they are used only once in one part of the code.

Tagging @PARAIPAN9

Reply:
seems like we both needed the same functionality at the same time, I'd say let's see which PR gets approved first and the other will change to the according method as soon as it's merged